src/main/modules/nunjucks/njkFilters.ts

```diff
@@ -12,6 +13,31 @@ function createFilters(env) {
     const fs = require('fs');
     const listTypes = publicationService.getListTypes();
     const languageLookup = JSON.parse(fs.readFileSync(path.resolve(__dirname, 'languageLookup.json')));
+    const rejectReasonLookup = JSON.parse(
+        fs.readFileSync(path.resolve(__dirname, 'media-account-rejection-reasons-lookup.json'))
+    );
+
+    // takes array or comma-separated string of rejection reason codes, transforms to an ordered list
+    // with optional padding depending on where it's being displayed.
+    env.addFilter('mediaRejectionClean', (csvList, nopadding = false) => {
+        csvList = Array.isArray(csvList) ? csvList : csvList.split(',');
+        const listItems = csvList
+            .map(x => rejectReasonLookup[x] ?? x)
+            .map(x => linkify(`<li><b>${x[0]}</b><br>${x[1]} </li>`))
+            .join('');
+        return new runtime.SafeString(`<ol${nopadding ? ' class="govuk-!-padding-left-4"' : ''}>${listItems}</ol>`);
+    });
+
+    // takes in a string and hunts for a url. If it finds one, it wraps it with link tags
+    // Note: Output from this will need to be used in a "safestring" to render properly.
+    function linkify(inputString) {
+        const urlRegex = /((https?:\/\/)[^\s]+)/g;
+        // target is blank to open in new tab
+        return inputString.replace(
+            urlRegex,
+            '<a href="$1" class="govuk-link" target="_blank">$1 (opens in a new tab)</a>'
```

Comment:
I dont think we should have the text "(opens in a new tab)" in a link. It is just additional text so should ideally be  outside the link. 

Also further down on the media rejection review screen, there is an option to open the image. That one mentions "open in a new window" and here is "open in a new tab". Can we make them consistent?

Reply:
Have done this now. I think you're right that it's better if it's open in a new window!